tests/snuba/tagstore/test_tagstore_backend.py

```diff
@@ -28,124 +24,120 @@ def setUp(self):
         self.ts = SnubaTagStorage()
 
         self.proj1 = self.create_project()
-        self.proj1env1 = self.create_environment(project=self.proj1, name="test")
-        self.proj1env2 = self.create_environment(project=self.proj1, name="test2")
-
-        self.proj1group1 = self.create_group(self.proj1)
-        self.proj1group2 = self.create_group(self.proj1)
```

Comment:
These tests needed to be rewritten to something closer to our actual ingest pipeline to work properly, which handles creating and updating groups, group hashes, event users, releases where necessary